<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createScenario</class>
 <widget class="QWidget" name="createScenario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Scenario :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_create">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cr√©er</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_scenario">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Ajouter phase</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choix du scenario :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_scenario">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de phase :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_phase">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Puissance :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Temps :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nb_power">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>20</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="nb_time">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>50</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tab_db">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_scenario</tabstop>
  <tabstop>btn_create</tabstop>
  <tabstop>combo_scenario</tabstop>
  <tabstop>txt_phase</tabstop>
  <tabstop>nb_power</tabstop>
  <tabstop>nb_time</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>tab_db</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
